### ISSUE TITLE
Fields of type array render empty HTML documentation

### ISSUE THREAD

**Issue by manoswerts** [asyncapi/generator#124]
### Steps to reproduce
1. Create a simple AsycnAPI specification with a field of type array:
```
asyncapi: '2.0.0-rc1'
id: 'urn:hello-world-app'
info:
  title: Hello world application
  version: '0.1.0'
channels:
  hello:
    subscribe:
      message:
        payload:
          type: array
          items:
            type: string
            pattern: '^hello .+$'
```
2. Generate HTML documentation from this spec
```
docker run --rm -it -v $PWD:/app -w /app asyncapi/generator ag hello.yaml html
```

### Expected result
The HTML documentation is generated correctly, showing the new field.

### Actual result
The HTML page generated is as good as empty (no error in the terminal during generation):
```
<html><head></head><body>null</body></html>
```

Markdown generation does seem to work with array properties (although the representation could be more clear).

**Comment by manoswerts** [comment]
I see it was closed at #110, but it still doesn't work for me.

**Comment by fmvilas** [comment]
Sorry, forgot to push Docker image. It's now on Docker Hub. Let me know if it works.
